addons/copy-message-link/addon.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Copy link button",
+  "description": "Adds \"Copy Link\" button to comments. Button copies link to comments, which you can use to show someone a comment.",
+  "credits": [{
+    "name": "Sly-Little-Fox",
+    "link": "https://github.com/Sly-Little-Fox/"
+  }],
+  "userscripts": [{
+      "url": "profile.js",
+      "matches": ["https://scratch.mit.edu/users/*", "https://scratch.mit.edu/studios/*"],
+      "runAtComplete": false
+    },
+    {
+      "url": "project.js",
+      "matches": ["https://scratch.mit.edu/projects/*"],
+      "runAtComplete": false
+    }
+  ],
+  "info": [{
+    "type": "notice",
+    "text": "Preferred button place will be ignored if it is not possible to place it button there.",
+    "id": "notice"
```

Comment:
Ideally this notice wouldn't be needed at all? Is it actually impossible to always place the button next to the report button?